src/stages/stage2/spells/amulet_of_harm.c

```diff
@@ -50,8 +50,8 @@ TASK(amulet_fire_spinners, { BoxedEnemy core; BoxedProjectileArray *spinners; })
 
 		ENT_ARRAY_FOREACH(ARGS.spinners, Projectile *p, {
 			int cnt = difficulty_value(12, 16, 22, 24);
-			for(int i = 0; i < cnt; ++i) {
-				cmplx ca = circle_dir(i, cnt);
+			for(int x = 0; x < cnt; ++x) {
+				cmplx ca = circle_dir(x, cnt);
```

Comment:
For reference, `gcc` complained that `int i = 0` (etc) was already used above, and it wanted a different var name.